<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-IS6DLKP" failures="2" tests="2" name="Test" time="0.097" errors="0" timestamp="2023-07-27T15:53:14 IST">
  <testcase classname="vtiger.Practice.assertionPraction" name="sampleTest" time="0.024">
    <failure type="java.lang.AssertionError" message="expected [2] but found [1]">
      <![CDATA[java.lang.AssertionError: expected [2] but found [1]
at vtiger.Practice.assertionPraction.sampleTest(assertionPraction.java:15)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- sampleTest -->
  <testcase classname="vtiger.Practice.assertionPraction" name="sampleTest1" time="0.006">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [true] but found [false],
	expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [true] but found [false],
expected [true] but found [false]
at vtiger.Practice.assertionPraction.sampleTest1(assertionPraction.java:37)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- sampleTest1 -->
</testsuite> <!-- Test -->
